source/PlayerInfo.cpp

```diff
@@ -4009,6 +4009,11 @@ void PlayerInfo::CreateMissions()
 		}
 	}
 
+	// Ensure missions are sorted respecting the order field.
+	// This is required so the correct `minor` missions are discarded first, and also orders priority or normal ones.
+	// Delaying the sort for the hasPriorityMissions case would be a tad faster, but IMHO simplicity trumps here.
```

Comment:
I'd drop this commentary since it's largely irrelevant, and we don't usually have personal commentary ("IMHO") in comments.